src/metatensor/models/cli/train_model.py

```diff
@@ -172,6 +174,9 @@ def train_model(
         if continue_from is None:
             continue_from = "null"
 
+        if not output.endswith(".pt"):
+            output = f"{output}.pt"
```

Comment:
I think we should warn when we do this. Similar to [metatensor](https://github.com/lab-cosmo/metatensor/blob/015d6d6eeef03ed4ab27f9d8a3a77e2f54356da9/python/metatensor-core/metatensor/io.py#L405-L411) where we do something like 

```python
    if isinstance(file, (str, pathlib.Path)):
        if not file.endswith(".npz"):
            file += ".npz"
            warnings.warn(
                message=f"adding '.npz' extension, the file will be saved at '{file}'",
                stacklevel=1,
            )
```

and maybe this check should go in the export function?

Reply:
Ok, will be done